storm/conf.yaml.example

```diff
@@ -0,0 +1,27 @@
+init_config:
+  # Default environment to use for all instances
+  # environment: production
+  #
+  # Default Storm API endpoint (typically this is the UI server on the UI port)
+  #   ex. if storm ui is running on the localhost on port 9005
+  # server: http://localhost:9005
```

Comment:
Is it interesting for the user to have multiple instances pointing to `localhost:9005` (not an expert on storm) ? If not lets remove this from `init_config` and force it to be set on each `instance`.

Reply:
agreed, will remove from defaults.